src/test/rustdoc/issue-55364.rs

```diff
@@ -0,0 +1,85 @@
+// ignore-tidy-linelength
+
+// First check the four top level modules are present
+// @has issue_55364/index.html
+// @has - '//section[@id="main"]/table//tr[@class="module-item"]/td/a[@class="mod"][@href="subone/index.html"]' 'subone'
```

Comment:
I think that checking if the modules are present is a bit too much. :)

Reply:
Fair, I'll take these out.